src/store.h

```diff
@@ -37,46 +37,54 @@ using Timestamp = time_point;
 class MetadataStore {
  public:
   struct Metadata {
-    Metadata(const std::string& version_,
+    Metadata(const std::string& type_,
+             const std::string& location_,
+             const std::string& version_,
+             const std::string& schema_name_,
              bool is_deleted_,
-             const Timestamp& created_at_,
              const Timestamp& collected_at_,
              json::value metadata_)
-        : version(version_), is_deleted(is_deleted_), created_at(created_at_),
-          collected_at(collected_at_), metadata(std::move(metadata_)),
-          ignore(false) {}
+        : type(type_), location(location_), version(version_),
+        schema_name(schema_name_), is_deleted(is_deleted_),
```

Comment:
Please keep the original indentation here.

Reply:
done